apps/anoma_node/test/supervisor_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule Anoma.SupervisorTest do
```

Comment:
We don't write tests in the codebase, see the style guide if you want to contribute here